src/common/metrics/location_summary.ts

```diff
@@ -9,7 +9,7 @@ const HIGH_NAME = 'Critical';
 const UNKNOWN = 'Unknown';
 
 const LEGEND_SUMMARY_LOW = 'On track to contain COVID';
-const LEGEND_SUMMARY_MEDIUM = 'Controlled disease growth';
+const LEGEND_SUMMARY_MEDIUM = 'Slow disease growth';
```

Comment:
I don't see this in the copy doc, can you double-check?

Reply:
yep! it was suggested by nirav and +1'ed by debbie. ill confirm with debbie

Reply:
confirmed!